jwst/pipeline/align_refs.cfg

```diff
@@ -1,2 +1,4 @@
-name = "alignrefs"
+name = "align_refs"
 class = "jwst.coron.AlignRefsStep"
+
+smoothing_box_length = 4
```

Comment:
Hidden developer information that is not clear: Since we're in the process of migrating from `cfg` files to crds-delivered step parameter files, all default specifications have been removed from the `cfg` files. Having the file itself with with the `name` and `class` are fine, but nothing else.

Reply:
How do I get this into the crds step parameter files?

Reply:
The fact that in the `spec` the default is defined as 4, there is no need to specify it in the cfg file.  If one wanted to change from the default, it can be added to their local copy of the cfg file. But, as delivered in the package, there is no reason to have defaults specified. This will be true for step parameter files, if and when they are ever created by INS.

Reply:
I've updated the config files and removed the try block.